recipes/xcube/meta.yaml

```diff
@@ -0,0 +1,87 @@
+{% set name = "xcube" %}
+{% set version = "0.2.0.dev1" %}
+
+package:
+  name: "{{ name }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://github.com/dcs4cop/{{ name }}/archive/{{ version }}.tar.gz"
+  sha256: "2617a04fb38511b9bba629e57c5358c4c2cb2384ec216334dd6f6947b43a80a3"
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+  entry_points:
+    # Put any entry points (scripts to be generated automatically) here.
+    - xcube = xcube.cli.cli:main
+    - xcube-gen = xcube.cli.gen:main
+    - xcube-restime = xcube.cli.restime:main
+
+requirements:
+  build:
+    # Python
+    - python =3.7
+    - pip =19.2
+  run:
+    # Python
+    - python =3.7
+    # Required
+    - affine =2.2
+    - blas =*=mkl
+    - click =7.0
+    - cmocean =2.0
+    - dask =1.2
+    - fiona =1.8
+    - gdal =2.4
+    - matplotlib =3.0
```

Comment:
Please depend on `matplotlib-base` when you don't need the Qt frontend explicitly.

Reply:
> Please depend on `matplotlib-base` when you don't need the Qt frontend explicitly.

Great, this way we get rid of the Qt-backend?!

Reply:
Yes, this saves quite some megabytes for end-users.

Reply:
Changed